File: src/test/java/com/indiepost/utils/ReadSystemEnvironmentTests.java
package com.indiepost.utils;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class ReadSystemEnvironmentTests {

    @Test
    public void signingSecretShouldReturnProperly() {
        String signingSecret = System.getenv("AUTH0_SECRET");
        assertThat(signingSecret).isNotNull();
    }
}

File: src/test/kotlin/com/indiepost/utils/ReadSystemEnvironmentTests.kt
package com.indiepost.utils

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class ReadSystemEnvironmentTests {

    @Test
    fun signingSecretShouldReturnProperly() {
        val signingSecret = System.getenv("AUTH0_SECRET")
        assertThat(signingSecret).isNotNull()
    }
}
